.github/workflows/checks.yml

```diff
@@ -0,0 +1,37 @@
+---
+name: Ersilia-Linter
+
+# Run this workflow on every PR to master
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  Setup:
+    runs-on: ubuntu-latest
```

Comment:
not sure if this repos maintainers plan to maintain for latest ubuntu or for a specific version. from the docker file it seems to be using ubuntu 20.04. 
@miquelduranfrigola  : can you confirm against which image the test should be run ?

Reply:
Hi! ubuntu-latest is good in my opinion!